kolibri/core/discovery/serializers.py

```diff
@@ -3,20 +3,19 @@
 from rest_framework.serializers import ValidationError
 
 from .models import ConnectionStatus
-from .models import NetworkLocation
 from .models import PinnedDevice
+from .models import StaticNetworkLocation
 from .utils.network import errors
 from .utils.network.client import NetworkClient
 from kolibri.core.serializers import HexOnlyUUIDField
 
 
 class NetworkLocationSerializer(serializers.ModelSerializer):
     class Meta:
-        model = NetworkLocation
+        model = StaticNetworkLocation
         fields = (
             "id",
             "available",
-            "dynamic",
```

Comment:
Thanks for addressing this issue with `dynamic` in the API response. I noticed I had put confusing and incorrect guidance in the issue.